src/shm/provider/alloc_layout.rs

```diff
@@ -62,19 +62,22 @@ pub extern "C" fn z_alloc_layout_new(
     alloc_layout_new(this, provider, size, alignment)
 }
 
+/// @warning This API has been marked as unstable: it works as advertised, but it may be changed in a future release.
 /// Constructs Alloc Layout in its gravestone value.
 #[no_mangle]
 pub extern "C" fn z_internal_alloc_layout_null(this_: &mut MaybeUninit<z_owned_alloc_layout_t>) {
     this_.as_rust_type_mut_uninit().write(None);
 }
 
+/// @warning This API has been marked as unstable: it works as advertised, but it may be changed in a future release.
 /// Returns ``true`` if `this` is valid.
 #[no_mangle]
 pub extern "C" fn z_internal_alloc_layout_check(this_: &z_owned_alloc_layout_t) -> bool {
     this_.as_rust_type_ref().is_some()
 }
 
-/// Borrows Alloc Layout
+/// @warning This API has been marked as unstable: it works as advertised, but it may be changed in a future release.
+/// Borrows Alloc Layout.
```

Comment:
Can you add @brief here, otherwise, this line will be merged with @warning above.

Reply:
fixed